src/github/SyftGithubAction.ts

```diff
@@ -126,7 +126,7 @@ async function executeSyft({
     throw new Error("Invalid input, no image or path specified");
   }
 
-  args = [...args, "-o", format];
+  args = [...args, "-o", format, "-o", "table"];
```

Comment:
I don't think this is going to work correctly. The way the code currently works, syft is invoked and the stdout captured as the sbom. Looking at the changes in the snapshots, this change seems to have caused multiple types of SBOMs to be output to stdout. Instead, we'd need to do something like `syft -o table -o spdx-json=some-file.spdx.json` and then instead of using stdout as the sbom, just print stdout to the console as this would have the table output, and then read or otherwise use the sbom file.

Reply:
Oh I see :) I got it really wrong.